features/step_definitions/networking.rb

```diff
@@ -595,23 +595,27 @@
   }
 end
 
-
 Given /^the multus is enabled on the cluster$/ do
   ensure_admin_tagged
-
   desired_multus_replicas = daemon_set('multus', project('openshift-multus')).replica_counters(user: admin)[:desired]
   available_multus_replicas = daemon_set('multus', project('openshift-multus')).replica_counters(user: admin)[:available]
```

Comment:
seems 600-601 is not useful now?  
other lgtm

Reply:
@zhaozhanqi Oops, forgot those duplicate lines, delete those now.

New Log from bad multus status cluster:
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/Runner-v3-smoke/4089/console

New Log from good multus status cluster:
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/Runner-v3-smoke/4091/console